qiita_pet/templates/study_ajax/process_study.html

```diff
@@ -0,0 +1,13 @@
+<script type="text/javascript">
```

Comment:
Where is this file used? And, it only includes javascript, why is not in a javascript file rather than in an HTML?